src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -91,7 +101,7 @@ export class EditorGroupModel extends Disposable {
 	private readonly _onDidActivateEditor = this._register(new Emitter<EditorInput>());
 	readonly onDidActivateEditor = this._onDidActivateEditor.event;
 
-	private readonly _onDidOpenEditor = this._register(new Emitter<EditorInput>());
+	private readonly _onDidOpenEditor = this._register(new Emitter<EditorOpenEvent>());
```

Comment:
A test for the correct properties of this new event would be nice: `editorGroupModel.test.ts`

Reply:
Added